cheatsheets_draft/OAuth_Cheat_Sheet.md

```diff
@@ -1,10 +1,10 @@
-# OAuth Cheat Sheet
+# Introduction
```

Comment:
The first header should be the CS name, where only one h1 should exist. The introduction should be `## Introduction`, after `#OAuth Cheat Sheet`

Reply:
Thanks, I will follow this structure!